test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -118,3 +124,45 @@ def _test_mnist_distributed(
         output = predictor.predict(data)
 
         assert output.shape == (batch_size, 10)
+
+        #  Check for Cloudwatch logs
+        if verify_logs:
+            _check_for_cloudwatch_logs(endpoint_name)
+
+def _check_for_cloudwatch_logs(endpoint_name):
+    client=boto3.client('logs')
+    identifyLogStream = client.describe_log_streams(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,    
+        orderBy='LastEventTime',
+        descending=True,
+        limit=5
+    )
+
+    logStreamName=identifyLogStream['logStreams'][0]['logStreamName']
```

Comment:
Is an alternative but efficient way to retrieve logStreamName than indexing? This can lead to an IndexError if identifyLogStream['logStreams'] is None.

Reply:
I thought of enclosing this in a try block. Bear in mind we do orderby last event time so if the describe_log_streams fails, then it can cause the indexing error like you mentioned, but that will fail the test and in this case I believe its ok? For now I am going to handle that exception and throw relevant information in the exception log